javadoc-parser/src/main/java/com/github/chhorz/javadoc/JavaDocParserBuilder.java

```diff
@@ -29,49 +33,90 @@
  */
 public class JavaDocParserBuilder {
 
-	private static final String BASE_INLINE_PATTERN = "\\{@%s ([^\\{\\}]+)\\}([\\s.,:;-])?";
-	private static final String INLINE_SUMMARY_PATTERN = String.format(BASE_INLINE_PATTERN, "summary");
-	private static final String INLINE_CODE_PATTERN = String.format(BASE_INLINE_PATTERN, "code");
-	private static final String INLINE_LINK_PATTERN = String.format(BASE_INLINE_PATTERN, "link");
-	private static final String INLINE_LINKPLAIN_PATTERN = String.format(BASE_INLINE_PATTERN, "linkplain");
-	private static final String INLINE_LITERAL_PATTERN = String.format(BASE_INLINE_PATTERN, "literal");
-	private static final String INLINE_VALUE_PATTERN = String.format(BASE_INLINE_PATTERN, "value");
-
 	private final JavaDocParser javaDocParser;
 
-	private JavaDocParserBuilder() {
-		this.javaDocParser = new JavaDocParser();
+	public static final List<BlockTag> STANDARD_JAVADOC_TAGS = Arrays.asList(new AuthorTag(),
```

Comment:
Why there not all javadoc tags?

Reply:
Because of the different handling of block and inline tags.